yass-components-react/src/components/Box/stories.tsx

```diff
@@ -0,0 +1,196 @@
+      control: { type: "select" },
+      options: classNames["border-color"],
+    },
+    borderStyle: {
+      control: { type: "select" },
+      options: classNames["border-style"],
+    },
+    borderRadius: {
+      control: { type: "select" },
+      options: classNames["border-radius"],
+    },
+    overflow: {
+      control: { type: "select" },
+      options: classNames["overflow"],
+    },
+    opacity: {
+      control: { type: "select" },
+      options: classNames["opacity"],
+    },
+    zIndex: {
+      control: { type: "select" },
+      options: classNames["z-index"],
+    },
+    textColor: {
+      control: { type: "select" },
+      options: classNames["color"],
+    },
+  },
+} as ComponentMeta<typeof Box>;
+
+const DefaultTemplate: ComponentStory<typeof Box> = ({
+  children,
+  paddingBlock,
+  paddingInline,
+  borderColor,
+  backgroundColor,
+  borderStyle,
+  borderWidth,
+  borderRadius,
+  overflow,
+  opacity,
+  zIndex,
+  // @ts-ignore TODO: Better story types
+  textColor,
+}) => (
+  <Box
+    paddingBlock={paddingBlock}
+    paddingInline={paddingInline}
+    borderColor={borderColor}
+    backgroundColor={backgroundColor}
+    borderStyle={borderStyle}
+    borderWidth={borderWidth}
+    borderRadius={borderRadius}
+    overflow={overflow}
+    opacity={opacity}
+    zIndex={zIndex}
+  >
+    <Text color={textColor}>{children}</Text>
+  </Box>
+);
+
+export const Default = DefaultTemplate.bind({});
+Default.args = {
+  children: "Some content in a box",
+  paddingBlock: "padding:10",
+  paddingInline: "padding:5",
+  borderColor: "border-color:blue-400",
+  backgroundColor: "background-color:blue-200",
+  borderStyle: "border-style:solid",
+  borderWidth: "border-width:2",
+  borderRadius: "border-radius:3",
+  overflow: "overflow:initial",
+  opacity: "opacity:1000",
+  zIndex: "z-index:100",
+};
+
+const colors = [
+  "neutral",
+  "red",
+  "orange",
+  "yellow",
+  "green",
+  "teal",
+  "blue",
+  "purple",
+  "pink",
+];
+const values = [
+  "50",
+  "100",
+  "200",
+  "300",
+  "400",
+  "500",
+  "600",
+  "700",
+  "800",
+  "900",
+];
+
+export const Colors: ComponentStory<typeof Box> = () => (
+  <Stack gap="gap:2">
+    {colors.map((color: string) => (
+      <Inline gap="gap:2">
+        {values.map((value: string) => (
+          <Box
+            padding="padding:20"
+            backgroundColor={`background-color:${color}-${value}`}
+          ></Box>
+        ))}
+      </Inline>
+    ))}
+  </Stack>
+);
+
+export const Opacity: ComponentStory<typeof Box> = () => (
+  <Inline gap="gap:10">
+    {classNames["opacity"].map((value: string) => (
+      <Stack alignItems="align-items:center">
+        <Text>{value}</Text>
+
+        <Box
+          backgroundColor="background-color:neutral-900"
+          padding="padding:30"
+          opacity={value}
+        ></Box>
+      </Stack>
+    ))}
+  </Inline>
+);
+
+export const BorderRadius: ComponentStory<typeof Box> = () => (
```

Comment:
I'm not a super large fan of this story, since everything beyond `border-radius:40` renders identically. But I can't think of a better way to visualise these
 
<img width="374" alt="image" src="https://user-images.githubusercontent.com/4335944/205898753-210f7199-25c0-4d6f-b479-9541c65c9a87.png">